python/tests/operations/one_hot.py

```diff
@@ -0,0 +1,90 @@
+import numpy as np
+import pytest
+
+import equistore
+from equistore import Labels
+
+
+class TestOneHot:
```

Comment:
Sorry I missed this on the previous review rounds. You don't need a class here, the test functions can be defined directly in the file.